src/components/payouts/payout-details/index.tsx

```diff
@@ -75,88 +64,95 @@ export const PayoutDetails: React.FC<{ memberId: string }> = ({ memberId }) => {
     })
   }
 
-  return (
-    <Formik
-      initialValues={{ category: '', amount: '', note: '', referenceId: '' }}
-      onSubmit={(data: any, { resetForm }) => {
-        if (!confirmed) {
-          return
-        }
+  const onSubmitHandler = (data: FieldValues) => {
+    if (!confirmed) {
+      setConfirmed((currentConfirmed) => !currentConfirmed)
+      return
+    }
```

Comment:
The latest "UX" is to use a `window.confirm` and not the double click functionality:
```suggestion
    if (!window.confirm(`Are you sure you want to payout ${data.amount} ${data.currency}?`)) {
      return
    }
```